grype/presenter/models/document.go

```diff
@@ -20,10 +21,18 @@ type Document struct {
 }
 
 // NewDocument creates and populates a new Document struct, representing the populated JSON document.
-func NewDocument(id clio.Identification, packages []pkg.Package, context pkg.Context, matches match.Matches, ignoredMatches []match.IgnoredMatch, metadataProvider vulnerability.MetadataProvider, appConfig any, dbInfo any, strategy SortStrategy) (Document, error) {
-	timestamp, timestampErr := time.Now().Local().MarshalText()
-	if timestampErr != nil {
-		return Document{}, timestampErr
+func NewDocument(id clio.Identification, packages []pkg.Package, context pkg.Context, matches match.Matches, ignoredMatches []match.IgnoredMatch, metadataProvider vulnerability.MetadataProvider, appConfig any, dbInfo any, strategy SortStrategy, outputTimestamp bool) (Document, error) {
+	var timestamp []byte
+
+	if !outputTimestamp || os.Getenv("GRYPE_TIMESTAMP") == "false" {
```

Comment:
All environment vars should be resolved up in the `cmd` package and not down in the library (it seems like this is already accounted for).
```suggestion
	if !outputTimestamp {
```